### PR TITLE
Tic tac toe

### PR DESCRIPTION
**Pull request by qpuhuk**
TicTacToe

### CODE REVIEW

File: src/main/java/AlexLink/HomeWork/HW1/Task3.java [hunk 1]
```diff
@@ -1,31 +1,28 @@
 package AlexLink.HomeWork.HW1;
 
-/*Имеется 4500 секунд. Вывести в консоль содержащихся в этом количестве секунд:
-А) минут + секунд,
-В) часов + минут + секунд,
-С) дней + часов + минут + секунд,
-D) недель + дней + часов + минут + секунд.
-по аналогии с примером выше.*/
+/*������� 4500 ������. ������� � ������� ������������ � ���� ���������� ������:
```
Comment (qpuhuk) [inline src/main/java/AlexLink/HomeWork/HW1/Task3.java]: Something happened with encoding

File: src/main/java/AlexLink/HomeWork/Arrays/Task1.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+//затем в столбик (отделяя один элемент от другого началом новой строки). Перед
+//созданием массива подумайте, какого он будет размера.
+//1) Найти произведение элементов, кратных 3.
+
+public class Task1 {
+
+    public static void main(String[] args) {
+        int[] evenNumbers = new int[10];
+        int sub = 1;
+        for (int i = 0, count = 0; i < evenNumbers.length; i++, count = count + 2) {
```
Comment (svetamolchanova) [inline src/main/java/AlexLink/HomeWork/Arrays/Task1.java]: can use count += 2

File: src/main/java/AlexLink/HomeWork/Arrays/Task1.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            evenNumbers[i] = count + 2;
+
+        }
+        for (int a : evenNumbers) {
+            System.out.print(a + " ");
+        }
+        System.out.println("\n");
+        for (int a : evenNumbers) {
+            System.out.print(a + "\n");
```
Comment (svetamolchanova) [inline src/main/java/AlexLink/HomeWork/Arrays/Task1.java]: can use System.out.println(a);

File: src/main/java/AlexLink/HomeWork/Arrays/Task9.java [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+public class Task9 {
+    static int sumLeftHalf;
+    static int sumRightHalf;
+    static int[] array = new int[0];
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Введите число: ");
+        if (scanner.hasNextInt()) {
```
Comment (svetamolchanova) [inline src/main/java/AlexLink/HomeWork/Arrays/Task9.java]: when you use "if", you need restart system, if you use "while", user have several attempts

File: src/main/java/AlexLink/HomeWork/HW1/Task3.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        int quantityIntMin = quantitySecAll % 60;
+        int quantityMinInSec = (quantitySecAll - quantityIntMin) / 60;
+        int quantityMin = quantityMinInSec % 60;
+        int quantityHoursOfSec = (quantityMinInSec - quantityMin) / 60;
+        int quantityHours = quantityHoursOfSec % 24;
+        int quantityDaysOfSec = (quantityHoursOfSec - quantityHours) / 24;
+        int quantityDays = quantityDaysOfSec % 7;
+        int quantityWeeks = (quantityDaysOfSec - quantityDays) / 7;
+
+        System.out.printf("%d ����� %d ������\n", quantityMin, quantityIntMin);
```
Comment (svetamolchanova) [inline src/main/java/AlexLink/HomeWork/HW1/Task3.java]: strange coding

File: src/main/java/AlexLink/HomeWork/HW3/Task16Metodichka.java [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+//        минимальным и максимальным значениями.
+
+public class Task16Metodichka {
+    private static int sum;
+
+    public static void main(String[] args) {
+
+        int[] array = {4, 7, 9, 33, 634, 64, 234, 31};
+        int min = 10;
+        int max = 100;
```
Comment (svetamolchanova) [inline src/main/java/AlexLink/HomeWork/HW3/Task16Metodichka.java]: why do you st these values? you need find min and max value in array

File: src/main/java/AlexLink/HomeWork/HW3/Task2.java [hunk 7]
```diff
@@ -0,0 +1,13 @@
+package AlexLink.HomeWork.HW3;
+
+//2)Одноклеточная амеба каждые 3 часа делится на 2 клетки. Определить,сколько амеб будет через 3, 6, 9, 12,..., 24 часа.
+
+public class Task2 {
+    public static void main(String[] args) {
+        int countAmoeba = 1;
+        for (int i = 3; i <= 24; i = i + 3) {
```
Comment (svetamolchanova) [inline src/main/java/AlexLink/HomeWork/HW3/Task2.java]: can use i += 3

File: src/main/java/AlexLink/HomeWork/HW1/Task3.java (hunk 1 above)
Comment (qpuhuk) [reply src/main/java/AlexLink/HomeWork/HW1/Task3.java]: correct

File: src/main/java/AlexLink/HomeWork/HW1/Task3.java (hunk 5 above)
Comment (qpuhuk) [reply src/main/java/AlexLink/HomeWork/HW1/Task3.java]: correct

**Review by qpuhuk — COMMENTED** [review]
correct

File: src/main/java/AlexLink/HomeWork/HW3/Task2.java (hunk 7 above)
Comment (qpuhuk) [reply src/main/java/AlexLink/HomeWork/HW3/Task2.java]: correct

File: src/main/java/AlexLink/HomeWork/Arrays/Task9.java (hunk 4 above)
Comment (qpuhuk) [reply src/main/java/AlexLink/HomeWork/Arrays/Task9.java]: correct

File: src/main/java/AlexLink/HomeWork/Arrays/Task1.java (hunk 2 above)
Comment (qpuhuk) [reply src/main/java/AlexLink/HomeWork/Arrays/Task1.java]: correct

File: src/main/java/AlexLink/HomeWork/Arrays/Task1.java (hunk 3 above)
Comment (qpuhuk) [reply src/main/java/AlexLink/HomeWork/Arrays/Task1.java]: correct

**Review by svetamolchanova — APPROVED** [review]

**Pull request merged**